ocean-client/addons/command-center/src/commands/set-pair.ts

```diff
@@ -0,0 +1,51 @@
+import { Bot } from '../utils/available-bot'
+import { fetchListOfPoolPairs } from '../utils/helpers'
+import { Command, Commands } from './command'
+
+export class SetPair extends Command {
+  private pair: string | undefined
+  private listOfPairs: string[] = []
+  private static usageMessage: string = Commands.SetPair + ' BTC-DFI'
+
+  static description = 'Sets given pair as new lm-reinvest pair.\n' + SetPair.usageMessage
+
+  static descriptionFor(bots: Bot[]): string | undefined {
+    if (!bots.includes(Bot.REINVEST)) return undefined
+    return SetPair.description
+  }
+
+  async prepare() {
+    this.listOfPairs = await fetchListOfPoolPairs()
+  }
+
+  availableFor(): Bot[] {
+    return [Bot.REINVEST]
+  }
+
+  parseCommandData(): void {
+    if (this.commandData.length === 2) {
+      this.pair = this.commandData[1]
+    }
+  }
+
+  validationErrorMessage(): string {
+    return 'Input parameter failed validation. Please use following\n' + SetPair.usageMessage
+  }
+
+  validate(): boolean {
+    if (this.pair === undefined) {
+      console.log('validate failed')
+      console.log('pair: ' + this.pair)
```

Comment:
pair is always undefined here. not much sense in logging it. better log commandData?